README.md

```diff
@@ -52,8 +52,8 @@ local grafana = ((import 'grafana/grafana.libsonnet') + {
                  }).grafana;
 
 k.core.v1.list.new(
-  grafana.dashboardDefinitions +
   [
+    grafana.dashboardDefinitions,
```

Comment:
Last time I checked lists couldn't be nested, which is why we did what we did. Has this changed recently?